src/services/mail-service.ts

```diff
@@ -0,0 +1,42 @@
+import config from 'config';
```

Comment:
this service needs to be redesigned and made more adaptable. To do this, you can make a factory, which will be initialized at the same time as the start of the application, read a configuration file and select the desired class of email service and return it instantiated. All of this should be made singles.

That is, the factory exemplars will implement an abstract email service with its methods. One, because Dima chose Sendgrid, let's call it that, and the second one be Dummy.

Dummy will be used in local development, tests and it is possible that the first time on production